tests/tests/optionsTests.robot

```diff
@@ -0,0 +1,85 @@
+
+Find and open Exclude temporary files
+    [Documentation]    Click Options, then toggle on/off Exclude temporary files switch button and verify if Reload Badge appears/disappears
+    [Tags]    options
+
+    Click Element    ${OptionsButton}
+    Wait Until Element Is Visible    ${OptionsList}
+    Click Element    ${ExcludeTemporaryFilesSwitch}
+    Wait Until Element Is Visible    ${ReloadBadge}
+    Click Element    ${ExcludeTemporaryFilesSwitch}
+    Wait Until Element Is Not Visible    ${ReloadBadge}
+
+Find and open Output logs to a file
+    [Documentation]    Click Options, then toggle on/off Output logs to a file button and verify if Reload Badge appears/disappears
+    [Tags]    options
+
+    Click Element    ${OptionsButton}
+    Wait Until Element Is Visible    ${OptionsList}
+    Wait Until Element Is Visible    ${LogsButton}
+    Click Element    ${OutputLogsSwitch}
+    Wait Until Element Is Visible    ${DownloadLogsButton}
+    Wait Until Element Is Not Visible    ${ReloadBadge}
+
+Find and open Extraction language filter
+    [Documentation]    Click Options, then Extraction language filter option, select every option and verify if Reload Badge appears/disappears
+    [Tags]    option
+
+    Click Element    ${OptionsButton}
+    Wait Until Element Is Visible    ${OptionsList}
+
+    Select From List By Index    extractionLanguageFilterOptions    0
+    List Selection Should Be    extractionLanguageFilterOptions    lang-plus-agn
+    Wait Until Element Is Not Visible    ${ReloadBadge}
+    Select From List By Index    extractionLanguageFilterOptions    1
+    List Selection Should Be    extractionLanguageFilterOptions    all
+    Wait Until Element Is Visible    ${ReloadBadge}
+    Select From List By Index    extractionLanguageFilterOptions    2
+    List Selection Should Be    extractionLanguageFilterOptions    lang
+    Wait Until Element Is Visible    ${ReloadBadge}
+    Select From List By Index    extractionLanguageFilterOptions    3
+    List Selection Should Be    extractionLanguageFilterOptions    lang-agn
+    Wait Until Element Is Visible    ${ReloadBadge}
+
+Find and open Collision resolution option
+    [Documentation]    Click Option, then Collision resolution, select every option and verify if Reload Badge appears/disappears
+    [Tags]    options
+
+    Click Element    ${OptionsButton}
+    Wait Until Element Is Visible    ${OptionsList}
+    Select From List By Index    collisionResolutionOptions    0
+    List Selection Should Be    collisionResolutionOptions    overwrite
+    Select From List By Index    collisionResolutionOptions    1
+    List Selection Should Be    collisionResolutionOptions    rename
+    Select From List By Index    collisionResolutionOptions    2
+    List Selection Should Be    collisionResolutionOptions    rename-all
+    Select From List By Index    collisionResolutionOptions    3
+    List Selection Should Be    collisionResolutionOptions    error
```

Comment:
It does appear, that you are not checking for reload badge here

Reply:
done